src/CoreWCF.RabbitMQ.Client/src/CoreWCF.RabbitMQ.Client.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
isn't netstandard2.0 a possible target ?

Reply:
This project references some apis made available in `System.ServiceModel.Primitives, version 6.0.0-preview1.23060.3`, which only supports net6.0 and above

Reply:
Even if System.ServiceModel.Primitives still supported netstandard2.0, it still couldn't. The problem is when targeting netstandard, you can end up running on .NET Framework. On Framework the Binding and BindingElement classes have references to IChannelListener for the service side of things, but these don't exist in the client. This would cause runtime problems as the implementation classes won't have the correct overrides of these methods. You won't even be able to instantiate an instance.  
The only way to do it would be to not target netstandard but multi-target .NET Framework and .NET 6.0. If there's demand, we can do that and just have throwing stubs for the service side methods and use conditional compilation. I don't think this is a big issue as the migration path would be .NET Framework WCF with MSMQ -> .NET Framework CoreWCF with MSMQ -> .NET CoreWCF with MSMQ -> .NET CoreWCF with RabbitMQ. If you have a substantial service, the migration will want to be done more gradually.